app/components/primer/issue_label_component.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Primer
+  # Use IssueLabel to add labels for issues and pull requests.
+  class IssueLabelComponent < Primer::Component
+    DEFAULT_VARIANT = :default
+    VARIANT_MAPPINGS = {
+      DEFAULT_VARIANT => "",
+      :big => "IssueLabel--big"
+    }.freeze
+
+    # @example auto|Schemes
+    #   <%= render(Primer::IssueLabelComponent.new(text: :white, bg: :blue)) { "Primer" } %>
+    #   <%= render(Primer::IssueLabelComponent.new(text: :white, bg: :red, scheme: :gray)) { "bug 🐛<" } %>
+    #   <%= render(Primer::IssueLabelComponent.new(text: :white, bg: :pink, scheme: :dark_gray)) { "help wanted" } %>
+    #   <%= render(Primer::IssueLabelComponent.new(text: :white, bg: :yellow, scheme: :yellow)) { "🚂 deploy: train" } %>
+    #
+    # @example auto|Variants
+    #   <%= render(Primer::IssueLabelComponent.new(text: :white, bg: :blue)) { "Primer" } %>
+    #   <%= render(Primer::IssueLabelComponent.new(text: :white, bg: :red, scheme: :gray)) { "bug 🐛<" } %>
+    #   <%= render(Primer::IssueLabelComponent.new(text: :white, bg: :pink, scheme: :dark_gray)) { "help wanted" } %>
```

Comment:
Could we updated the `Label` docs to point at this component so if folks land there and want this functionality, they can find their way here?